client/matchmaking/matchmaking-disabled-card.tsx

```diff
@@ -62,15 +43,12 @@ const CountdownItemText = styled.span`
   color: ${colorTextSecondary};
 `
 
-export interface ConnectedMatchmakingDisabledCardProps {
+export interface DisabledCardProps {
```

Comment:
This rename seems fairly confusing given that there is now a separate component called `DisabledCard` and these are not props for it